pandas/core/frame.py

```diff
@@ -1911,26 +1915,32 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
 
         See Also
         --------
-        pandas.read_stata : Import Stata data files
-        pandas.io.stata.StataWriter : low-level writer for Stata data files
-        pandas.io.stata.StataWriter117 : low-level writer for version 117 files
+        pandas.read_stata : Import Stata data files.
+        pandas.io.stata.StataWriter : Writer for Stata data files.
+        pandas.io.stata.StataWriter117 : Writer for version 117 files.
 
         Examples
         --------
-        >>> data.to_stata('./data_file.dta')
+
+        >>> df = pd.DataFrame({'col1': [1, 2, 3], 'col2': [4, 5, 6]})
+        >>> df.to_stata('./data_file.dta')
 
         Or with dates
 
-        >>> data.to_stata('./date_data_file.dta', {2 : 'tw'})
+        >>> dates_1 = pd.to_datetime(["2016-05-06","2017-05-07", "2018-05-10"])
+        >>> dates_2 = pd.to_datetime(["2016-11-06","2017-11-07", "2018-11-10"])
+        >>> df_dates = pd.DataFrame({'date1': dates_1, 'date2': dates_2})
+        >>> df_dates.to_stata('./date_data_file.dta', {2 : 'tw'})
 
         Alternatively you can create an instance of the StataWriter class
 
-        >>> writer = StataWriter('./data_file.dta', data)
+        >>> StataWriter = pd.io.stata.StataWriter
+        >>> writer = StataWriter('./data_file.dta', df)
```

Comment:
I'd use instead (assuming you're using the animals data):
`>>> writer = pd.io.stata.StataWriter('animals.dta', df)`